Cargo.toml

```diff
@@ -53,7 +52,7 @@ packed_simd = { version = "0.3.0", features = ["into_bits"], optional = true }
 [build-dependencies]
 rand = { version = "0.5", default-features = false }
 byteorder = { version = "^1.2.3", default-features = false, features = ["i128"] }
-digest = "0.7"
+digest = { version = "0.8", default-features = false }
 generic-array = "0.9"
```

Comment:
Should this `generic-array` dependency also be removed?

Reply:
Indeed. Removed.

Reply:
Err, too late!